src/virtac/atip_server.py

```diff
@@ -70,6 +70,9 @@ def __init__(
             limits_csv (str): The filepath to the .csv file from which to
                                     load the pv limits, for more information
                                     see create_csv.py.
+            bba_csv (str): The filepath to the .csv file from which to
```

Comment:
At some point, we will have to assume a certain directory structure and drop the endless arguments. Not for now, though